walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -45,89 +46,96 @@
       }
     }
```

Comment:
If you feel like learning something new, I recommend trying to do this with java streams. It is often more clear than for loops.

https://stackoverflow.com/questions/44180101/in-java-what-are-the-advantages-of-streams-over-loops
https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html